src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { TodoInfo } from '../../types/TodoInfo';
 
-export const TodoModal: React.FC = () => {
+type TodoModalProps = {
+  todoInfo: TodoInfo | null;
+  isLoadTodoModal: boolean;
+  onCloseTodo: () => void;
+};
+
+export const TodoModal: React.FC = ({
```

Comment:
The `TodoModal` component is defined as a `React.FC` but does not specify the props type. You should define it as `React.FC<TodoModalProps>` to ensure type safety and proper type checking.